pkg/volume/iscsi/iscsi.go

```diff
@@ -97,6 +97,11 @@ func (plugin *iscsiPlugin) newBuilderInternal(spec *volume.Spec, podUID types.UI
 	lun := strconv.Itoa(iscsi.Lun)
 	portal := portalBuilder(iscsi.TargetPortal)
 
+	iface := iscsi.ISCSIInterface
+	if iface == "" {
+		iface = "default"
```

Comment:
Not sure about that. This is already an implied parameter in open-iscsi, if no -I parameter is provided, assume -I default.  If you can point me to a preferred way to do this, I would be happy to change the code but removing this force default to "default" requires adding checks for this parameter for every call to iscsiadm, this approach seemed much easier.